.github/workflows/ci.yml

```diff
@@ -0,0 +1,109 @@
+name: "Build and deploy"
+
+on:
+  push:
+
+jobs:
+  publish:
+    name: Publish to GitHub Pages
+    needs: pdf
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/master'
+
+    steps:
+    - name: Download website build
+      uses: actions/download-artifact@v2
+      with:
+        name: website
+        path: public/
+
+    - name: Download PDF build
+      uses: actions/download-artifact@v2
+      with:
+        name: pdf
+        path: pdf/
+
+    - name: Move PDF files
+      run: mv pdf public/
+
+    - name: Deploy to GitHub Pages
+      uses: peaceiris/actions-gh-pages@v3
+      if: github.ref == 'refs/heads/master'
```

Comment:
needed when you have the condition at the job level?

Reply:
Nope, you're right, I forgot to delete this line when moving the condition to the job level. I'll get rid of it. Thanks for flagging.